### PR TITLE
LPS-172083 Create Payment Confirmation and kebab fragment button

### PR DESCRIPTION
**Pull request by EuniceTravassos**

### CODE REVIEW

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by Claudionilton** [comment]
Just starting reviewing :)

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-button-save/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+/* eslint-disable @liferay/portal/no-global-fetch */
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-button-save/index.js:2]: Remove line 1 and 2.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
+{
+	"label": {
+		"en_US": "EVP Payment Confirmation"
+	},
+	"name": "EVPPaymentConfirmation",
+	"objectActions": [
+	],
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json:7]:
```suggestion
	
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+			"DBType": "Date",
+			"businessType": "Date",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Payment Date"
+			},
+			"listTypeDefinitionExternalReferenceCode": "",
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json:19]:
```suggestion
			
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+			"name": "paymentDate",
+			"objectFieldSettings": [
+			],
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json:22]:
```suggestion
			
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+			"system": false,
+			"type": "Long"
+		}
+	],
+	"objectLayouts": [
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "control_panel.object",
+	"parameterRequired": false,
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json:92]:
```suggestion
	
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+	"pluralLabel": {
+		"en_US": "EVP Payment Confirmations"
+	},
+	"scope": "company",
+	"system": false,
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/invoice-request.json:97]:
```suggestion
	
```

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Pull request closed without merging**